src/main/java/org/folio/rmapi/model/CustomLabel.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.rmapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+public class CustomLabel {
+  
+  @JsonProperty("id")
+  private Integer id;
+  @JsonProperty("displayLabel")
+  private String displayLabel;
+  @JsonProperty("displayOnFullTextFinder")
+  private Boolean displayOnFullTextFinder;
+  @JsonProperty("displayOnPublicationFinder")
+  private Boolean displayOnPublicationFinder;
+  
+  @JsonProperty("id")
```

Comment:
I think it's sufficient to only annotate fields with @JsonProperty, no need to annotate getter and setters